addons/analytic/models/analytic_account.py

```diff
@@ -123,6 +123,12 @@ def name_get(self):
             res.append((analytic.id, name))
         return res
 
+    def copy(self, default=None):
+        default = dict(default or {})
+        if not default.get('name'):
+            default['name'] = _("%s (copy)") % (self.name)
+        return super().copy(default)
```

Comment:
You could even fit everything on one line (maybe a bit less readable though)
```suggestion
        return super().copy_data({'name': _("%s (copy)", self.name), **(default or {})})
```